recipe/build.sh

```diff
@@ -94,8 +145,6 @@ if [ $(uname) == Darwin ]; then
                 -headerdir $PREFIX/include/qt \
                 -archdatadir $PREFIX \
                 -datadir $PREFIX \
-                -L $PREFIX/lib \
-                -I $PREFIX/include \
```

Comment:
Why this removal?

Reply:
With the new way that compiler flags are set in this PR, these flags are no longer necessary — they're subsumed into the modifications of the mkspec files.

Reply:
Ok, thanks for the explanation.